dicebot/handlers/message/long_message_handler.py

```diff
@@ -4,7 +4,7 @@
 from dicebot.data.types.message_context import MessageContext
 from dicebot.handlers.message.abstract_handler import AbstractHandler
 
-LONG_MESSAGE_CHAR_THRESHOLD = 700
+LONG_MESSAGE_CHAR_THRESHOLD = 1000
```

Comment:
Fair

Reply:
Haha I was wondering if you'd notice that